package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsonwebtokenydoo",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Since `jsonwebtokenydoo` doesn't have any tests at the moment, do you think it'd be best to publish this as a major (potentially backwards-incompatible) version?